.github/workflows/kind.yml

```diff
@@ -928,12 +928,11 @@ jobs:
       - name: Deploy Antrea
         run: |
           kubectl apply -f build/yamls/antrea.yml
+          kubectl rollout status --timeout=1m deployment/antrea-controller -n kube-system
+          kubectl rollout status --timeout=1m daemonset/antrea-agent -n kube-system
```

Comment:
It seems the agentAvailableTimeout is 5mins, but here is using 1m here, is this expected?